search/2016-09-01/swagger/searchservice.json

```diff
@@ -0,0 +1,3797 @@
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/IndexerExecutionInfo"
+            }
+          }
+        }
+      }
+    },
+    "/indexes": {
+      "post": {
+        "tags": [
+          "Indexes"
+        ],
+        "operationId": "Indexes_Create",
+        "description": "Creates a new Azure Search index.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/rest/api/searchservice/Create-Index"
+        },
+        "parameters": [
+          {
+            "name": "index",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Index"
+            },
+            "description": "The definition of the index to create."
+          },
+          {
+            "name": "client-request-id",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "format": "uuid",
+            "description": "The tracking ID sent with the request to help with debugging.",
+            "x-ms-client-request-id": true,
+            "x-ms-parameter-grouping": { "name": "search-request-options" }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "x-ms-request-id": "request-id",
+        "responses": {
+          "201": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/Index"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Indexes"
+        ],
+        "operationId": "Indexes_List",
+        "description": "Lists all indexes available for an Azure Search service.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/rest/api/searchservice/List-Indexes"
+        },
+        "parameters": [
+          {
+            "name": "$select",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Selects which properties of the index definitions to retrieve. Specified as a comma-separated list of JSON property names, or '*' for all properties. The default is all properties."
+          },
+          {
+            "name": "client-request-id",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "format": "uuid",
+            "description": "The tracking ID sent with the request to help with debugging.",
+            "x-ms-client-request-id": true,
+            "x-ms-parameter-grouping": { "name": "search-request-options" }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "x-ms-request-id": "request-id",
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/IndexListResult"
```

Comment:
@olydis Thanks for the tip. I had noticed that feature when updating our Management SDK. However, our data plane SDK is still on a much older version of AutoRest and we didn't have the time to upgrade (this involves changing a lot of hand-written code to fix bugs fixed in the generated code, etc.). The next time we bump the major version number, we'll make this change.